### DISCUSSION TITLE
How to format text in excalidraw ?

### DISCUSSION THREAD

**Discussion by Shiraz-V-Kamar** [excalidraw/excalidraw#5050] [Q&A]
Right now there is no option to give 

- **bold**
-  _italic_

while in excalidraw will that be possible?

I would also like to know if its possible to change the color of single word in a sentence.

**Comment by ad1992** [comment]
We don't support these as of now but surely in the discussion so hopefully sometime soon 🙂

**Reply by pranjalcodefire** [reply]
Any update on this?

**Reply by imajinl** [reply]
+1

**Reply by LordOkami** [reply]
+1

**Reply by connerdassen** [reply]
How is such a basic feature still not supported???

**Reply by dnnr1** [reply]
We need bold plz

**Reply by sheanhoxie** [reply]
> How is such a basic feature still not supported???

Because you haven't written it yet! lol 😉

**Reply by TobiasWinter97** [reply]
+1

**Reply by charlestbell** [reply]
+1

**Comment by Mrazator** [comment]
It's on the roadmap! Let's track it here https://github.com/excalidraw/excalidraw/issues/5138

**Comment by marcospgp** [comment]
almost 3 years and still can't make text bold :sweat:

**Reply by pauldorman** [reply]
Now more than 4 years. Grand Theft Auto VI will be out before the Excalidraw devs start listening to us.

**Comment by seancorc** [comment]
This is a pretty basic feature that would really be appreciated. New user who might not continue using the product because of this

**Reply by ilyadynin** [reply]
Yeah, I just discovered Excalidraw because I thought about mixing different types of notes in Obsidian, and really need handwritten notes/PDFs and Excalidraw works for that BUT, I really dont know if I want to continue using it as I cannot make bullet points, and make text bold, it's really all that I need...

**Reply by AdriGrana** [reply]
This is a common theme I've been noticing while trying to get into Excalidraw these past few weeks, I sometimes struggle to do very very basic things and the first thing I do is blame it on my lack of knowledge of using the tool, but after some googling (like I did now) I realize that this feature I thought as very basic isn't supported at all rather than me not finding out how to do it, which is quite a shock each time that happens. No bold text, no italics? I feel like that needs to be implemented before any other features, heck I can't even change the font size other than the three presets (unless I start using the zoom in/out functionallity which has other drawbacks), also I cannot change the font of a single character within a text box (only the entire text box). It's not detremental but I agree, I know people who would quit using this tool entirely as soon as realizing you cannot do these things I just mentioned.

**Comment by mattie47** [comment]
It's 2025. How is there no underline, bold, italics? 😭 😭 😭 

I also find it wild that I can modify font size with shortcut keys, but I can't see/set what value they are in the GUI. 

This is such a great tool - I just wish these quality of life improvements which feel so fundamental were there!

**Comment by abudawud** [comment]
August 2025, indeed we need this feature.
For now, i create two text and align it with the base text. and this very painful.

Hopefully this issue closed soon with that feature.

Thanks for the greate tool.

**Comment by talgatenergysphere** [comment]
I support the request to add support for text and paragraph formatting.

**Comment by aga-relation** [comment]
I can't stress how useful this feature would be 🙏  Any update on when we can expect it to drop?

**Comment by pauldorman** [comment]
I concur! I thought I'd try out Excalidraw for keeping some programming notes, but went back to Advanced Canvas within the 30 seconds it took me to discover there was no text formatting whatsoever. I was thinking highlighted code would look nifty in the default Excalidraw font too, so that would be nice also.

**Comment by xxiiss** [comment]
**Must have**

**Comment by KaiqueCovo** [comment]
Oh my God, 4 years and we don't have the basics 😞. Canceling my plan because we don't have **bold** and **highlight text**.

Going back to tldraw, which is free.
